tests/cypress/integration/newfold-labs/wp-module-onboarding/steps/basic-info.cy.js

```diff
@@ -16,6 +16,13 @@ describe('Basic Info Page', function () {
     it('Check if Drawer opened', () => {
         cy.get('.nfd-onboarding-drawer__panel-site-title-container').scrollIntoView().should('be.visible');
     });
+	
+    it('Checks Active Link is Basic Info', () => {
```

Comment:
Can we add 'in the Drawer Panel' or something to ​`Checks Active Link is Basic Info​`, for a better reference?